application/models/user.go

```diff
@@ -512,7 +513,21 @@ func (u *User) GetPhotoURL(tx *pop.Connection) (*string, error) {
 // Save wraps tx.Save() call to check for errors and operate on attached object
 func (u *User) Save(tx *pop.Connection) error {
 	u.Nickname = domain.RemoveUnwantedChars(u.Nickname, "-_ .,'&@")
-	return save(tx, u)
+	if err := save(tx, u); err != nil {
+		appError := api.AppError{Err: err}
+		if strings.Contains(err.Error(), "Nickname must have a visible character") {
+			appError.Key = "UpdateUser.InvisibleNickname"
```

Comment:
Should this be a const in the api package?

Reply:
Yep, thanks